starter/js/atm.js

```diff
@@ -1,43 +1,54 @@
 //Begin with the document ready function
+$(document).ready(function(){
 
+}
     //Checking account deposit function
-
       //On click of the depositChecking button
-
+$("#depositChecking").click(function(event){
         //Get value from the amountChecking input field
-
+   var amountBalance = $("#amountChecking").val();
         //Take that value and add it to the existing value in the checkingBalance div
-
+   var checkingBalance = amountBalance.val();
     //Checking account withdrawl funtion
-
+});
+event();
       //On click of the withdrawChecking button
-
+$("#withdrawChecking").click(function(event){
         //Get value from the amountChecking input field
-
+var amountBalance = $("#amountChecking").val();
         // If that value is greater than the value in the account ignore that action
         // In other words if this would put the account into a negative balance do not allow it
-
+if (amountBalance < 0) {
+  return prompt("Not allowed!");
+} else {
+  checkingBalance - amountBalance;
```

Comment:
Make sure to do something with the result of this subtraction. Set it to a variable? Use that variable and put it to the screen?

```js
var newBalance = checkingBalance - amountBalance
// do something with newBalance
```